checker_test.go

```diff
@@ -103,6 +104,47 @@ got:
 want:
   ~string "bar"~
 `),
+}, {
```

Comment:
It would be nice to have test cases for
- a string with a single newline at the end of the string
- a string with a single newline followed by some text

Reply:
The "string with a single newline at the end of the string" is already there. WIll add the string starting with newline case.